src/mjolnir/valhalla_add_elevation.cc

```diff
@@ -6,19 +6,22 @@
 #include <boost/property_tree/ptree.hpp>
 #include <cxxopts.hpp>
 
+#include "baldr/graphid.h"
 #include "baldr/graphreader.h"
 #include "baldr/graphtile.h"
 #include "baldr/rapidjson_utils.h"
+#include "config.h"
 #include "mjolnir/elevationbuilder.h"
-#include "mjolnir/valhalla_add_elevation_utils.h"
 
 namespace opt = cxxopts;
 
 using namespace valhalla::baldr;
 using namespace valhalla::midgard;
 using namespace valhalla::mjolnir;
 
-enum Input { CONFIG, TILES };
+boost::property_tree::ptree config;
+std::vector<std::string> tiles;
+uint32_t num_threads = 0;
```

Comment:
I'm kinda sorry for this whole refactor, it wasn't really necessary, but I felt a need for consistency with the other executables. If we have boilerplate code, I'd personally prefer it to be actual boilerplate:)